### DISCUSSION TITLE
Airflow webserver cannot start: error: invalid command 'webserver'

### DISCUSSION THREAD

**Discussion by nicnguyen3103** [apache/airflow#22343] [General]
### Apache Airflow version

2.2.4 (latest released)

### What happened

I was performing update for my swarm cluster from 2.2.2 to 2.2.4. After airflow db upgrade, both worker and scheduler can start just fine but the airflow webserver returns error: invalid command 'webserver'. 

I exec into the worker container and try to start the webserver manually from there to reproduce the issue. Here is the screenshot from the CLI of the worker container when I try to start the webserver there 

![airflow_webserver_log](https://user-images.githubusercontent.com/46189714/158810984-a1130600-09c5-4404-b40d-0b7ca10e36d0.png)

### What you think should happen instead

I have no idea how to fix this issue as the airflow webserver start but then stop to yield command error, which is quite weird. 

### How to reproduce

here is the command in the dockerfile for a clean airflow setup on my side

RUN . $AIRFLOW_VENV/bin/activate && pip install 'apache-airflow[mysql,amazon,celery,papermill,ssh]==2.2.4' --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.2.4/constraints-3.8.txt"
RUN . $AIRFLOW_VENV/bin/activate && pip install -r $HOME/requirements-extend-airflow.txt

with requirements-extend-airflow.txt contains 

`apache-airflow[statsd]
apache-airflow[sentry]
sentry-sdk==1.1.0
blinker==1.4
cached-property==1.5.1
mysqlclient==2.0.1
Flask-Bcrypt==0.7.1
Flask-OAuthlib==0.9.5
virtualenv==20.10.0`

### Operating System

Ubuntu "20.04.2 LTS (Focal Fossa)"

### Versions of Apache Airflow Providers

apache-airflow-providers-amazon==3.0.0
apache-airflow-providers-celery==2.1.0
apache-airflow-providers-ftp==2.0.1
apache-airflow-providers-http==2.0.3
apache-airflow-providers-imap==2.2.0
apache-airflow-providers-mysql==2.2.0
apache-airflow-providers-papermill==2.2.0
apache-airflow-providers-sqlite==2.1.0
apache-airflow-providers-ssh==2.3.0

### Deployment

Docker-Compose

### Deployment details

Remote logs on s3

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
It looks like you have SOME configuration problem. I doubt it comes from the Airflow, rather than it is either some of your customization or the effect of your extra installation steps. 

I think you should investigate further on what is causing your problem:

* do you have any special customisations in your image (env var/paramerers) 
* what does your `airflow info` command print.
* what happens if you build your image without the "extend requirements" line ? does it still fail to sstart?
* what happes if you remove the extra "airflow[]"  installs
* what is your airflow version reported after the installation

**Reply by potiuk** [reply]
FYI. It's not reproducible in a stock Airlfow "community" image even with your extensions - so you really need to dig deeper.
My suggestion is to disable your customizations one-by-one in your image and see which one causes the problem. 

Happy to help if you investigate and let us know which of those causes the problem, but I am unable to help more.

**Reply by nicnguyen3103** [reply]
many thanks for your suggestion, I am trying to remove package by package to troubleshoot the issue and will get back to you after I found which config broke my airflow.

**Reply by nicnguyen3103** [reply]
Hi @potiuk , many thanks again for your support. After carefully eliminating the package and investigating, I found that one of the newly installs custom plugins from my team broke the airflow command.  Removing the plugin and having a fresh install of airflow works! 

Have a wonderful day!

**Reply by potiuk** [reply]
Cool. Good to know ! Thanks for letting us know!
